src/cli.zig

```diff
@@ -368,7 +368,7 @@ pub const Arguments = struct {
             Global.exit(1);
         };
 
-        const print_help = args.flag("--help");
+        const print_help = args.flag("--help") or args.flag("-h");
```

Comment:
did you check if the `or args.flag("-h")` is needed. i think clap is able to do this implicitly once you speicfy the -h alias  in the param list